jscomps/EmbedLiteChromeManager.js

```diff
@@ -91,6 +91,14 @@ EmbedLiteChromeListener.prototype = {
       message["type"] = event.type;
       break;
     case "DOMPopupBlocked":
+      let permissions = Services.perms.getAllForURI(Services.io.newURI(event.target.URL, null, null));
+      while (permissions.hasMoreElements()) {
+        let permission = permissions.getNext().QueryInterface(Ci.nsIPermission);
+        if (permission.type == "popup" && permission.capability == Ci.nsIPermissionManager.DENY_ACTION) {
+          // Ignore popup
+          return;
```

Comment:
Thank you, it's nice!
Only one small questions about 
https://github.com/sailfishos/embedlite-components/pull/6/files#diff-3beb0dd2468be58ec089f9a69c809099bbcbe8abffe93d5fdd1040f569c10eb4R106
It can be used instead 102 line, I think as it used in cases before this, what do you think about it?
messageName = "embed:popupblocked"

Reply:
Yep, done.